### ISSUE TITLE
Internal recorder suggestion

### ISSUE THREAD

**Issue by Strata69** [absolllute/Mega-Hack-Pro-Future#925]
Building on issue #922, if you're familiar with how many slow-mo cameras work, it would be quite useful to have the system perpetually recording but overwriting old video past the length of say 5 minutes, so that you can just click a button and save the previous 5 minutes of recording or smth like that, i think the term is "loop recording"?

**Comment by Brittank88** [comment]
> i think the term is "loop recording"?

This is correct. OBS refers to it as a "Replay Buffer" which I think is a good way to explain it too.
